music21/harmony.py

```diff
@@ -580,11 +598,11 @@ class ChordStepModification(prebase.ProtoM21Object):
 
     # INITIALIZER #
 
-    def __init__(self, modType=None, degree=None, intervalObj=None):
-        self._modType = None  # add, alter, subtract
-        self._interval = None  # alteration of degree, alter ints in mxl
-        self._degree = None  # the degree number, where 3 is the third
-        # use properties if defined
+    def __init__(self, modType=None, degree=None, intervalObj=interval.Interval('P1')):
```

Comment:
Not so sure about this change because Interval is mutable. We definitely need an immutable flag on Interval so that this default value isn't ever transposed inPlace.

Reply:
Great catch! 34aa5da